ESCPOS_NET/Emitters/BaseCommandEmitter/CharacterCommands.cs

```diff
@@ -5,8 +5,36 @@ namespace ESCPOS_NET.Emitters
     public abstract partial class BaseCommandEmitter : ICommandEmitter
     {
         /* Character Commands */
+        /// <summary>
+        /// Select print mode(s)
+        /// </summary>
+        /// <param name="style"></param>
+        /// <returns></returns>
         public virtual byte[] SetStyles(PrintStyle style) => new byte[] { Cmd.ESC, Chars.StyleMode, (byte)style };
 
+        /// <summary>
+        /// Select character size. Please reference <see cref="https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=34"/>
+        /// Value from 0 to 7, 0 for normal size.
+        /// </summary>
+        /// <param name="widthMagnification">Enlargement in horizontal direction.</param>
+        /// <param name="heightMagnification">Enlargement in vertical direction.</param>
+        /// <returns></returns>
+        public virtual byte[] SetSize(byte widthMagnification, byte heightMagnification)
+        {
+            if (widthMagnification > 7) widthMagnification = 7;
+            if (heightMagnification > 7) heightMagnification = 7;
```

Comment:
Can we actually have enums for the magnification parameters? I see 7 is the maximum, but what is the minimum acceptable value for magnification?  Does magnification = 1 means it's a regular-sized print?

Perhaps we want to add some validation, like the ones that exist for Barcode commands, in order to prevent users to use out of range magnification values. What do you think?